css/style.css

```diff
@@ -1,4 +1,38 @@
-@import url(https://fonts.googleapis.com/css?family=Open+Sans:300,400,400italic,700);
+/*
+ * Muli is licensed under SIL OFL Version 1.1
+ * See /assets/logos/font/Muli/OFL.txt file for full information.
+ */
+@font-face {
+    font-family: 'Muli';
+    src: url('/assets/logos/font/Muli/Muli-Bold.woff') format('woff');
+    font-weight: 700;
```

Comment:
In my experience, on the web it's easier to just rely on `font-weight: bold` for font declarations rather than using values. At one point or another, someone will use a `bold` value, or a `strong` or `b` tag, and then it sucks (the fallback rules for 700 would work, but those rules are weird and it always ends up being a pain).

We'd be better with all 4 combinations of `font-weight: normal | bold` + `font-style: normal | italic`.

Reply:
Bold shorthand isn't guaranteed to be 700, and we don't serve other weights, so we're better off with numbers imo

Reply:
What do you mean by "isn't guaranteed"? CSS has no clue what file is for what value, that's why we need to declare them like you're doing here. Numeric values are useful when you define several of them, like 500 to 900, but if you're defining only one (like we're doing here), then `bold` is our best bet. Really no benefits in using numeric values here.